README.md

```diff
@@ -661,306 +702,6 @@ Disabled as default. Set `SPACESHIP_EXIT_CODE_SHOW` to `true` in your `.zshrc`,
 | `SPACESHIP_EXIT_CODE_SYMBOL` | `✘` | Character to be shown before exit code |
 | `SPACESHIP_EXIT_CODE_COLOR` | `red` | Color of exit code section |
 
-### Example
```

Comment:
I agree that this is repeats options twice, but it's pretty useful though. It's easy to copy-paste them into config and tune what is needed. What about moving this section into the wiki page?

Reply:
Moving to wiki page may cause inconsistencies if we miss to update the wiki when options are added/modified. 

If we're going with wiki, It could be added either in presets page or to a new one. I personally prefer the presets page, For single point of access.

Reply:
How about providing a minimal example, which customizes the prompt order (remove some sections, switch the order of the rest) and then changes some symbols/colors of the remaining sections?

Reply:
Like this https://github.com/denysdovhan/spaceship-zsh-theme/pull/148#issuecomment-357451513 @nfischer ?

Reply:
@salmanulfarzy that's reasonable. But should also include changing a couple symbols and colors, since those are one-liners.

Reply:
Sure. Let's continue discussion on #313